lix/source-code/client/src/helpers.ts

```diff
@@ -18,7 +18,8 @@ export const withLazyFetching = ({
 }: Args): NodeishFilesystem => {
 	return new Proxy(nodeishFs, {
 		get(getTarget: typeof nodeishFs, prop, receiver) {
-			if (getTarget[prop as keyof typeof nodeishFs]) {
+			// eslint-disable-next-line no-prototype-builtins
+			if (((getTarget as any).__proto__ as any).hasOwnProperty(prop)) {
```

Comment:
not sure i want this in general, why is this needed except for your testing?

Reply:
@janfjohannes This problem occured when you subclass a nodish fs and mark a method as private - instead of the functional approach since private functions become visible to the target. The Proxy should only proxy methods that are defined in the NodishFs type but this is not the case with the current implementation.

Reply:
@martin-lysk  not sure what you mean, lets discuss in a call.